streampipes-model/src/main/java/org/apache/streampipes/model/staticproperty/RemoteOneOfStaticProperty.java

```diff
@@ -52,7 +52,7 @@ public RemoteOneOfStaticProperty(String internalName, String label, String descr
     this.valueFieldName = valueFieldName;
     this.labelFieldName = labelFieldName;
     this.descriptionFieldName = descriptionFieldName;
-    this.valueRequired = valueRequired;
+    this.optional = valueRequired;
```

Comment:
Does this have the right meaning here? Because if valueRequired=true, then optional should be false, right?
Do we have to rename this variable?

Reply:
@tenthe `RemoteOneOfStaticProperty` isn't used anymore and deprecated since 2019, I propose to open a separate PR after merging this one to delete the property completely. The change here is just due to automatic refactoring.